polyply/src/annotate_dna.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2020 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from random import sample
+from difflib import SequenceMatcher
+from itertools import permutations
+import numpy as np
+import networkx as nx
+from .processor import Processor
+
+base_library = {"DA": "DT", "DT": "DA", "DG": "DC", "DC": "DG",
```

Comment:
capitalize the dict as 'BASE_LIBRARY', this is a convention thing. Also I think it would be good if this dict could be somehow overwritten or made accessible by CLI. Like if people of custom base-pair names.